mm2src/coins/utxo/slp.rs

```diff
@@ -2018,12 +2033,17 @@ mod slp_tests {
             142, 135, 205, 228, 173, 0, 0, 0, 0, 0, 25, 118, 169, 20, 140, 255, 252, 36, 9, 208, 99, 67, 125, 106, 168,
             183, 90, 0, 155, 155, 165, 27, 113, 252, 136, 172, 216, 36, 92, 97,
         ];
-        let tx_bytes_str = hex::encode(tx_bytes);
 
+        let tx_bytes_str = hex::encode(tx_bytes);
         let err = fusd.send_raw_tx(&tx_bytes_str).wait().unwrap_err();
         println!("{:?}", err);
         assert!(err.contains("is not valid with reason outputs greater than inputs"));
 
+        let err2 = fusd.send_raw_tx_bytes(tx_bytes).wait().unwrap_err();
+        println!("{:?}", err2);
+        assert!(err2.contains("is not valid with reason outputs greater than inputs"));
+        assert_eq!(err, err2);
```

Comment:
I think it's enough to check if `err` just equals to `err2`

Reply:
I wanted to double check this to achieve more strict testing. For example, if previous assertion check and `send raw_tx_bytes` error gets updated by mistake, this check will fail the tests and save us from some bad moments. Even if it's so low possibility, I think we should have written tests for as many possibility as we can.